_emails/somerville.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Somerville, MA
+permalink: "/somerville"
+name: Letter to Council Members
+city: Somerville
+state: MA
+recipients:
+- cityclerk@somervillema.gov
+- citycouncil@somervillema.gov
+- mayor@somervillema.gov
+subject: Demanding support for defunding the police in Somerville
+body: |-
+  Dear Mayor Curtatone and Somerville City Councilors,
+
+  My name is \[YOUR NAME\], and I am a resident of \[CITY\]. This past week, our nation has been gripped by protests calling for a rapid and meaningful reconsideration of the role of policing in communities as well as an end to racism and anti-Blackness in America. Our city has been deeply engaged in much of this action. Accordingly, it has come to my attention that the city plans to adopt a number of the reform proposals created by Campaign Zero. I ask that you reconsider these attempts to reform and instead pivot your attention to defunding the police for fiscal year 2021.
+
+  SPD has been a waste of our resources. In fiscal year 2020, the SPD budget was approximately 18% of the total—less only than schools. The city of Somerville will spend over $17,044,021 on the Police Department this year, as you well know. By contrast, Health and Human Services receives only $3,064,691 and Culture & Recreation receives only $4.5 million (1.8% of the budget). Additionally, within the $21 million of grants received in 2018, $749,867 was allocated to the police system, while only $120,610 was allocated to the Arts Council and a mere $503,187 was allocated to Sustainability & Environment.
+
+  This is an abomination. Budgets for Immigrant services, disability services, and fair housing services also pale in comparison to this excessive budget. While we’ve been spending extraordinary amounts on policing, we have not seen improvements to safety, homelessness, mental health, or affordability in our city. Instead, we see wasteful and harmful actions of our police.
+
+  I call on you to defund the SPD and instead meaningfully reallocate funds towards social programs and resources that support housing, jobs, education, health care, child care, and other critical community needs. I demand a budget that supports community wellbeing, rather than empowers the police forces that tear them apart.
+
+  We can be a beacon for other cities to follow if only we have the courage to change.
+
+  Sincerely,
+
+  \[YOUR NAME\]
+  \[YOUR ADDRESS\]
+  \[YOUR EMAIL\]
+  \[YOUR PHONE NUMBER\]
```

Comment:
Please remove bracket escaping here, as this is not necessary for them to render correctly